### ISSUE TITLE
Linking libnettle to libgmp

### ISSUE THREAD

**Issue by mczarnek** [ShiftMediaProject/nettle#3]
Seems like every time I run project_get_dependencies.bat from scratch, I have to add a project reference from libnettle to libgmp in the ffmpeg_deps.sln before it will build.

Would be nice if there was some way to automatically add that reference when building. Not a big deal, please fix the gnutls and other issues first but just a would be nice kind of feature.

**Comment by Sibras** [comment]
Not sure what the issue would be. I checked and ffmpeg_deps.sln has the libgmp project dependency by default and running nettles project_get_dependencies.bat doesnt modify the project/solution so this is not something i can reproduce.

**Comment by mczarnek** [comment]
I apologize, this does seem to be fixed within the new update. Was previously an issue and I'd been doing it anyway to be safe and figured I'd go ahead and say something knowing you were in the process of updating. Should've checked first. Sorry.

**Comment by mczarnek** [comment]
Huh, no, I was wrong about that, it is still an issue.

I built everything from scratch(in MTd modeas noted under FFmpeg issue ticket.. shouldn't make a difference) and have 4ish projects fail until I added a reference from libnettle to libgmp. I see it being added as an additional dependency.

Is it possible that If you right click on the solution, then select "Common Properties-> ProjectDependencies" and choose libnettle as the project, that maybe it isn't marked as depending on libgmp, meaning they are built in the wrong order and the library doesn't exist by the time libnettle starts linking? Can't check on mine as I already added the dependency.
